git_repo/repo.py

```diff
@@ -586,9 +588,39 @@ def main(args):
         return 2
 
 
+class Version:
```

Comment:
This should be moved to either:

- [the tools module](https://github.com/guyzmo/git-repo/blob/devel/git_repo/tools.py) if left as is ;
- [the kwargparse module](https://github.com/guyzmo/git-repo/blob/devel/git_repo/kwargparse.py) if integrated to the kwargparse/dispatcher system
- its own new module otherwise (i.e.: as a new independant module)